cmd/osbuild-worker/config.go

```diff
@@ -68,10 +68,9 @@ type containersConfig struct {
 }
 
 type executorConfig struct {
-	Type            string `toml:"type"`
-	IAMProfile      string `toml:"iam_profile"`
-	KeyName         string `toml:"key_name"`
-	CloudWatchGroup string `toml:"cloudwatch_group"`
+	Type       string `toml:"type"`
+	IAMProfile string `toml:"iam_profile"`
+	KeyName    string `toml:"key_name"`
```

Comment:
Just thinking loud: this will cause TOML parser to fail if and only if TOML is parsed strict, not sure how upgrades will be done but deprecation could be safer.